borg/testsuite/archiver.py

```diff
@@ -71,6 +71,29 @@ def __exit__(self, *args, **kw):
                 os.environ[k] = v
 
 
+def exec_cmd(*args, archiver=None, fork=False, **kw):
```

Comment:
cmd() below is a instance method and it uses unittest methods. I wanted to have a function I can use everywhere and that does not depend on unittest or subclassing.